src/test/java/core/basesyntax/reportcreator/CsvReporterTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.reportcreator;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CsvReporterTest {
+    private static final CsvReporter csvReporter = new CsvReporter();
+    private static final List<String> VALID_DATA_LIST = new ArrayList<>(2);
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        VALID_DATA_LIST.add("banana,152");
+        VALID_DATA_LIST.add("apple,90");
+    }
+
+    @Test
+    public void testEmptyDataInput_Ok() {
+        String expected = "fruit,quantity\n";
```

Comment:
```suggestion
        String expected = REPORT_TITLE;
```
it repeats 3 times in the class, makes sense to declare it as a constant